java/PaimonCDCSink/README.md

```diff
@@ -0,0 +1,165 @@
+  "ApplicationDescription": "Sink CDC from Kafka as Apache Paimon table",
+  "RuntimeEnvironment": "FLINK-1_20",
+  "ServiceExecutionRole": "Your service role arn",
+  "ApplicationConfiguration": {
+    "ApplicationCodeConfiguration": {
+      "CodeContent": {
+        "S3ContentLocation": {
+          "BucketARN": "Your bucket arn",
+          "FileKey": "Your jar file s3 key"
+        }
+      },
+      "CodeContentType": "ZIPFILE"
+    },
+    "EnvironmentProperties": {
+      "PropertyGroups": [
+        {
+          "PropertyGroupId": "ActionConf",
+          "PropertyMap": {
+            "action": "kafka_sync_database",
+            "database": "Your Paimon Database",
+            "warehouse": "Your paimon warehouse path"
+          }
+        },
+        {
+          "PropertyGroupId": "KafkaConf",
+          "PropertyMap": {
+            "kafka_conf@_properties.bootstrap.servers": "MSK bootstrap servers",
+            "kafka_conf@_properties.auto.offset.reset": "earliest",
+            "kafka_conf@_properties.group.id": "group id",
+            "kafka_conf@_topic": "Your cdc topic",
+            "kafka_conf@_value.format": "debezium-json"
+          }
+        },
+        {
+          "PropertyGroupId": "CatalogConf",
+          "PropertyMap": {
+            "catalog_conf@_hadoop.fs.s3.impl": "org.apache.hadoop.fs.s3a.S3AFileSystem",
+            "catalog_conf@_hadoop.fs.s3.buffer.dir": "/var/tmp"
+          }
+        },
+        {
+          "PropertyGroupId": "TableConf",
+          "PropertyMap": {
+            "table_conf@_bucket": "4",
+            "table_conf@_metadata.iceberg.storage": "hive-catalog",
+            "table_conf@_metadata.iceberg.manifest-legacy-version": "true",
+            "table_conf@_metadata.iceberg.hive-client-class": "com.amazonaws.glue.catalog.metastore.AWSCatalogMetastoreClient",
+            "table_conf@_fs.s3.impl": "org.apache.hadoop.fs.s3a.S3AFileSystem",
+            "table_conf@_fs.s3.buffer.dir": "/var/tmp",
+            "table_conf@_sink.parallelism": "4"
+          }
+        }
+      ]
+    }
+  },
+  "FlinkApplicationConfiguration": {
+    "ParallelismConfiguration": {
+      "AutoScalingEnabled": true,
+      "Parallelism": 4,
+      "ParallelismPerKPU": 1
+    }
+  },
+  "CloudWatchLoggingOptions": [
+    {
+      "LogStreamARN": "arn:aws:logs:us-west-2:YourAccountId:log-group:/aws/kinesis-analytics/kafka-cdc-paimon:log-stream:kinesis-analytics-log-stream"
+    }
+  ]
+}
+```
+
+Last, create an MSF application using AWS CLI.
+
+```shell
+aws kinesisanalyticsv2 create-application \
+--cli-input-json file://create-kafkacdc-paimon.json
+```
+
+### Running in IntelliJ
+
+You can run this example directly in IntelliJ, without any local Flink cluster or local Flink installation.
+
+See [Running examples locally](../running-examples-locally.md) for details.
+
+### Generating data
+
+You can use [Kinesis Data Generator](https://github.com/awslabs/amazon-kinesis-data-generator),
```

Comment:
I think this is nor correct. The example uses Kafka.

To run this example e2e the user should set up a DB, and a CDC Kafka Connector. This is quite heavyweight, particularly to test the application locally.

Would it be possible to add a dockercompose the user can run to set up locally what required?

Also, doing CDC directly in Flink would simplify the requirements for running this example e2e, both locally and on AWS